### PR TITLE
Add a help page to the SXG validator popup.

### LINKED ISSUE #6: Add help doc / link in the extension popup
In the extension popup window, it would be useful to provide a link to a longer help doc with an explanation of all of the fields. I know what they are, but not everyone will. For example, what do the warning messages mean?

### PR DESCRIPTION
**Pull request by banaag**
Fixes #6

### CODE REVIEW

File: help.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+   ✅ means everything's ok.
+   <br>
+   ❌ means there's an error.
+   <br>
+  </div>
+</div>
+<table border="1" cellpadding="3" style="border-collapse:collapse;">
+  <tr>
+    <td style="min-width:100px" nowrap>URL</td>
+    <td align="left"><span id="url" alt="Origin URL">The URL of the SXG's origin, cache or SXG certificate.</span></td>
```
Comment (twifkak) [inline help.html:65]: Why `alt=` here? AFAIK that should be used for screenreaders (e.g. for image tags), but they can read the inner text just fine. (Ditto below.)

**Review by twifkak — APPROVED** [review]

File: help.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+body {
+  min-width: 100px; 
+  position: relative;
+  vertical-align:middle;
+}
+
+.label-container {
+  padding-top: 8px;
+  padding-bottom: 10px;
+  display: flex;
```
[the remark this answers is not in the document]
Comment (banaag) [reply help.html:12]: yeah that's needed. Removed.

File: help.html (hunk 1 above)
Comment (banaag) [reply help.html:65]: Cut & paste. Removed.

File: help.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (twifkak) [reply help.html:12]: Needed for `.label-container` but not for `.debug-container`? Still curious why, but no big deal anyway; just picking a nit.

**Review by twifkak — APPROVED** [review]

**Pull request merged**